### PR TITLE
namelists and run script for NGGPS 3km NH benchmark

### PR DESCRIPTION
**Pull request by mt5555**
[BFB]

### CODE REVIEW

**Comment by mt5555** [comment]
No impact on tests, so skipping integration

**Comment by oksanaguba** [comment]
I just noticed i had to review this, sorry for the delay. For SL namelists we should have nu_q=0 and semi_lagrange_nearest_point_lev=100.

**Comment by oksanaguba** [comment]
You switched to higher nu for ne256, so, it is good enough for zeta? I wasn't able to output ne256 with 3100x1600 grid on 80 anvil nodes, with 'pnetcdf'. How to do it?

**Comment by oksanaguba** [comment]
Can I add a trivial file to your PR? It is to detect multiple definitions of variables and to get diffs between two namelists. It will look something like below and it really helps tracking things.

```
file1=homme-template.nl
file2=homme-template-SL.nl
echo "Comparing namelists ${file1}, ${file2}"
diff <( sed 's/=/ = /' ${file1} | awk '/^[ \t]*[[:alnum:]]/ { print $1, $3 }' | sort ) <(sed 's/=/ = /' ${file2} | awk '/^[ \t]*[[:alnum:]]/ { print $1, $3 }' | sort )

echo "Now look for doubles in each file"
for file in ./*.nl
do
  echo "in file ${file}"
  sed 's/=/ = /' ${file} | awk '/^[ \t]*[[:alnum:]]/ { print $1 }' | sort | uniq -d && echo "Above are redeclared variables if any."
done
```

**Comment by oksanaguba** [comment]
This PR has vert remap fix, it is probably not meant to be here since it has a separate PR?

File: components/homme/test/benchmarks/NGGPS/nggps-ne256-sl.nl [hunk 1]
```diff
@@ -15,6 +15,7 @@ disable_diagnostics = .true.
 restartfreq   = 43200
 restartfile   = "./R0001"
 runtype       = 0
+theta_hydrostatic_mode=.true.
 tstep=40
```
Comment (oksanaguba) [inline components/homme/test/benchmarks/NGGPS/nggps-ne256-sl.nl]: this nl also needs theta_advect_form = 1 ?

**Comment by mt5555** [comment]
good point about the vert remap.  I'll rebase this branch once that makes it to master.

I'll make the change to the SL namelist.

**Pull request merged**